src/commands/whoami.js

```diff
@@ -0,0 +1,51 @@
+const Command = require('../base')
+const globalFlags = require('../helpers/global-flags')
+const authHelper = require('../helpers/auth')
+const requestHelper = require('../helpers/request')
+const ora = require('ora')
+const chalk = require('chalk')
+
+/**
+ * Tells you who you're logged in as
+ */
+class WhoamiCommand extends Command {
+  async run() {
+    if (!authHelper.isLoggedIn()) {
+      return this.log(`Not sure, you aren't logged in yet. Run ${chalk.yellow('chec login')} to continue.`)
```

Comment:
Isn't this a double-up? Or do you want the explicit message in this case?

Reply:
Yeah true, no reason this needs to be here